modules/gltf/doc_classes/GLTFDocumentExtension.xml

```diff
@@ -1,8 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="GLTFDocumentExtension" inherits="Resource" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../doc/class.xsd">
 	<brief_description>
+		[GLTFDocument] extension class.
 	</brief_description>
 	<description>
+		Extends the functionality of the [GLTFDocument] class by allowing you to run arbitrary code at various stages of GLTF import or export.
+		To use, make a new class extending GLTFDocumentExtension, override any methods you need, make an instance of your class, and register it using [method GLTFDocument.register_gltf_document_extension].
+		[b]Note:[/b] Like GLTFDocument itself, all GLTFDocumentExtension classes must be stateless in order to function properly. If you need to store data, use [member GLTFState.extra_data] or [member GLTFNode.extra_data].
```

Comment:
Thoughts on a better name that extra data. Maybe `extension_data`? Not sure.

Reply:
Actually I had that name first but I changed it because I didn't want to create confusion since both of these also have an `"extensions"` member that gets saved in the JSON.

Reply:
That makes sense.

Reply:
I just found out that GLTF "extras" are also a thing, so I renamed it again to `additional_data`.